images/cassandra/rootfs/initialize.sh

```diff
@@ -7,6 +7,7 @@ if [[ $NUM_NODES -gt 2 ]]; then
 	REPLICATION_FACTOR=3
 fi
 
-sed -i 's/REPLICATION_FACTOR/'"${REPLICATION_FACTOR}"'/g' /init.cql
+sed 's/REPLICATION_FACTOR/'"${REPLICATION_FACTOR}"'/g' /init.cql > /tmp/init.cql
+cat /tmp/init.cql > /init.cql # avoid issues with sed and writing in /
```

Comment:
I am not sure what are you trying to solve here with this step? Is there any problem with sed writing to read-only root? Why then the second command is working?

Reply:
Yes, otherwise it is confusing.

Reply:
added